.env.example

```diff
@@ -0,0 +1,11 @@
+# Your Twilio Account SID
```

Comment:
What do you think about adding a reference to the Twilio console? 
The required values comes from https://www.twilio.com/console